mne/epochs.py

```diff
@@ -343,13 +343,15 @@ def preload_data(self):
         assert self._data.shape[-1] == len(self.times)
         return self
 
-    def decimate(self, decim, copy=False):
+    def decimate(self, decim, jitter=0, copy=False):
         """Decimate the epochs
 
         Parameters
         ----------
         decim : int
             The amount to decimate data.
+        jitter : int
+            Apply a jitter to where the decimation starts
```

Comment:
`jitter` is not the right term here, you probably mean `offset`. You should mention that it's in samples at the original sampling rate.